src/Adapter/Module/Module.php

```diff
@@ -257,6 +257,18 @@ public function onInstall()
         return $result;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function onPostInstall(): bool
```

Comment:
Could you test it ?

Reply:
I can't test it, this class rely on Object model with `LegacyModule` :(